pos_order_return/tests/test_pos_order_return.py

```diff
@@ -116,6 +118,7 @@ def setUp(self):
         res = self.pos_order.action_pos_order_invoice()
         self.invoice = self.env["account.move"].browse(res["res_id"])
 
+    @unittest.skip("Errors when other OCA / Pos modules are installed")
```

Comment:
Out of curiosity, why were these test passing with travis ?

Reply:
I got it ! 

Travis became red very recently.

https://github.com/OCA/pos/commit/3b955a12895cdd6a1c72ce34aaaf684c6c859a2b was green.
https://app.travis-ci.com/github/OCA/pos/jobs/557872233#L1609

https://github.com/OCA/pos/commit/f7131a15bdfb2ebdc95d1efe7019a33dde91b9ec was red : 
https://app.travis-ci.com/github/OCA/pos/jobs/564379816#L1606

![image](https://user-images.githubusercontent.com/3407482/161261477-4220aca5-fa9a-4eb5-baaa-b17a10f2492e.png)

the only changes are translation, so that's not because of recent OCA changes.

I tested again ``locally`` with an updated odoo core, and the module doesn't work now. So something changed in Odoo core. (point of sale or stock), that makes the tests failing.
I updated the text of the "skip" part.

@dsolanki-initos : for your information, the module you migrated (https://github.com/OCA/pos/pull/671) is now failing.